assets/styles/callouts.scss

```diff
@@ -0,0 +1,350 @@
+:root {
+    --dark: #141021;
+    --outlinegray: #dadada;
+    --summary: #00b0ff;
+    --summary-accent: #7fd7ff;
+    --bug: #f50057;
+    --bug-accent: #ff7aa9;
+    --danger: #ff1744;
+    --danger-accent: #ff8aa1;
+    --example: #7c4dff;
+    --example-accent: #bda5ff;
+    --fail: #ff5252;
+    --fail-accent: #ffa8a8;
+    --info: #00b8d4;
+    --info-accent: #69ebff;
+    --note: #448aff;
+    --note-accent: #a1c4ff;
+    --question: #64dd17;
+    --question-accent: #b0f286;
+    --quote: #9e9e9e;
+    --quote-accent: #cecece;
+    --done: #00c853;
+    --done-accent: #63ffa4;
+    --important: #00bfa5;
+    --important-accent: #5fffe9;
+    --warning: #ff9100;
+    --warning-accent: #ffc87f;
```

Comment:
can these be prefixed with `callout` so e.g. `--callout-warning`

Reply:
Fixed